packages/generators/app/lib/resources/templates/database-templates/js/database.template

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+
+module.exports = ({ env }) => {
+	const client = env('DATABASE_CLIENT', 'sqlite')
+
+	const connections = {
+		mysql: {
```

Comment:
it looks like the mysql and postgres config are the same ?

Reply:
Yes. 

However postgres has way more options in the `ssl` config. I focused on the ones that were in common as I didn't knew which is relevant. I don't think we need to have every possible option available.

These options should (hopefully) cover the majority of cases.